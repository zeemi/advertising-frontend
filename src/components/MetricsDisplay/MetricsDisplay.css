.MetricsDisplay {
    display: grid;
    grid-template-columns: repeat(3, 1fr);
    grid-template-rows: 1fr;
    grid-column-gap: 15px;
    grid-row-gap: 15px;
}

.MetricsDisplay .filters {
    grid-area: 1 / 1 / 2 / 2;
    box-shadow: rgba(0, 0, 0, 0.1) 0px 1px 6px, rgba(0, 0, 0, 0.1) 0px 1px 4px;
    padding: 5px;
    background-color: aliceblue;

}

.MetricsDisplay .chart {
    grid-area: 1 / 2 / 2 / 4;
    padding: 5px;
    box-shadow: rgba(0, 0, 0, 0.1) 0px 1px 6px, rgba(0, 0, 0, 0.1) 0px 1px 4px

}