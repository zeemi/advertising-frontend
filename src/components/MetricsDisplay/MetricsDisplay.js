import React from 'react';
import Chart from '../Chart/Chart';
import Filters from '../Filters/Filters';

const MetricsDisplay = () => {
  return <React.Fragment>
    <Chart/>
    <Filters/>
  </React.Fragment>
};

export default MetricsDisplay;