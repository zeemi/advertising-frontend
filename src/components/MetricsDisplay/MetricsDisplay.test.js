import React from 'react';
import {shallow} from 'enzyme';
import MetricsDisplay from './MetricsDisplay';

describe('<MetricsDisplay>', function () {
  it('should render filters and chart', function () {
    expect(shallow(<MetricsDisplay/>)).toMatchSnapshot()
  });
});