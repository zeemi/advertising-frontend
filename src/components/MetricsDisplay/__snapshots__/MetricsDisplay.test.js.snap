// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MetricsDisplay> should render filters and chart 1`] = `
<Fragment>
  <Chart />
  <Filters />
</Fragment>
`;
